src/Root.cpp

```diff
@@ -159,6 +159,14 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 	m_MojangAPI = new cMojangAPI;
 	bool ShouldAuthenticate = settingsRepo->GetValueSetB("Authentication", "Authenticate", true);
 	m_MojangAPI->Start(*settingsRepo, ShouldAuthenticate);  // Mojang API needs to be started before plugins, so that plugins may use it for DB upgrades on server init
+	AString MaxLoadedChunks = settingsRepo->GetValueSet("Server", "MaxChunkRAM", "30");
+	m_MinimizeRam = settingsRepo->GetValueSetB("Server", "MinimizeRAM", true);
+	if (!StringToInteger(MaxLoadedChunks, m_MaxRAMChunks))
+	{
+		settingsRepo->SetValue("Server", "MaxChunkRAM", "30");
+		m_MaxRAMChunks = 30;  // in MiB
```

Comment:
@madmaxoft @tigerw The motive behind 30 per server:

- For a server with few players, which is typically the case, the server enjoys caching without eating too much RAM. This covers most people.
- For a server with many players, the server uses the "agressive" strategy, which is just like Vanilla's strategy. It is less optimal. but is not that bad and definitely better than master.
- For a server with many players, the owner can dedicate more RAM and get the caching back. This should preferably be done for most dedicated servers.

The choice of 30 was arbitrary. It seemed like a sensible value which of "not too much RAM", taking Raspberry Pis into account.